clients/hadoopfs/README.md

```diff
@@ -0,0 +1,7 @@
+# hadoop-lakefs file system
```

Comment:
Not sure this is helpful without explaining some more targets and options.  Also not needed on this PR?

Reply:
Will remove